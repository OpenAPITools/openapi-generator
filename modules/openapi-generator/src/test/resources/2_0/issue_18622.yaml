swagger: '2.0'
info:
  title: 'Buggy Api'
  version: '1.0'
consumes:
  - application/json
paths:
  /foo/bar:
    post:
      responses:
        '200':
          description: ok
          schema:
            $ref: "#/definitions/myresponse"
definitions:
  myresponse:
    type: object
    additionalProperties: false
    properties:
      x:
        $ref: "#/definitions/myenum"
  myenum:
    type: string
    enum:
      - A
      - B
      - C
