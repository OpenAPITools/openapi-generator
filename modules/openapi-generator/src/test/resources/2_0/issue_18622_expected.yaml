openapi: 3.0.1
info:
  title: Buggy Api
  version: "1.0"
servers:
  - url: /
paths:
  /foo/bar:
    post:
      responses:
        "200":
          content:
            '*/*':
              schema:
                $ref: '#/components/schemas/myresponse'
          description: ok
components:
  schemas:
    myresponse:
      additionalProperties: false
      example:
        x: A
      properties:
        x:
          $ref: '#/components/schemas/myenum'
      type: object
    myenum:
      enum:
        - A
        - B
        - C
      type: string
x-original-swagger-version: "2.0"
